biz_portal/apps/portal/templates/portal/business_detail.html

```diff
@@ -83,7 +83,7 @@
               <img class="status-icon" src="{% static 'images/ellipse.svg' %}" />
             </span>
             <span class="type-header">Status:</span>
-            <span class="mdc-expansion__header">{{ object.registration_status.label }}</span>
+            <span class="mdc-expansion__header">{% if object.registration_status %} {{ object.registration_status.label }} {% else %} Status unknown {% endif %}</span>
```

Comment:
In future you could use `firstof` which is much cleaner https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#firstof